spec/classes/datadog_agent_spec.rb

```diff
@@ -197,7 +197,7 @@
         it do
           is_expected.to contain_package('Datadog Agent').with(
             ensure: 'installed',
-            install_options: ['/norestart', { 'APIKEY' => 'notakey', 'HOSTNAME' => 'notahost', 'TAGS' => '""', 'NPM' => 'true' }],
+            install_options: ['/norestart', { 'APIKEY' => 'notakey', 'HOSTNAME' => 'notahost', 'TAGS' => '""', 'ADDLOCAL' => 'MainApplication,NPM' }],
```

Comment:
You'll have to open up the file above, but the `let` statement above implies to me this is only A7.
There's no reason it won't work in A6, does there need to be another change for that?

Reply:
This is just the test. It only covers A7, but the code is the same for both Agents so I don't think we need to test both.